components/upload/Upload.tsx

```diff
@@ -47,6 +47,7 @@ const InternalUpload: React.ForwardRefRenderFunction<unknown, UploadProps> = (pr
     style,
     itemRender,
     maxCount,
+    propagateDropEvent = true,
```

Comment:
Naming seems should to be better. 🤔

Reply:
any suggestion? 😸

Reply:
No thoughts, please ignore above.

Reply:
It might make more sense to name this something like `propagateDragDropEvents` since it affects multiple drag/drop events, not just the drop event.